divyatag/update-index (#28726)
DataDog/documentation: content/en/agent/fleet_automation/_index.md

After you enable Remote Configuration on an Agent, you can send a flare from Datadog. For instructions on sending a flare, see [Send a flare from the Datadog site][7].

> When contacting Datadog Support with Remote Configuration enabled for an Agent, our Support team may initiate a flare from your environment in order to better assist you in a timely manner. Flares provide troubleshooting information to our support team to help you resolve your issue.

Reviewer: ```suggestion
When you contact Datadog Support with Remote Configuration enabled for an Agent, the Support team may initiate a flare from your environment. Flares provide troubleshooting information to Datadog Support.
```
